Foundation/NSAttributedString.swift

```diff
@@ -7,9 +7,14 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+import CoreFoundation
 
 public class NSAttributedString : NSObject, NSCopying, NSMutableCopying, NSSecureCoding {
     
+    private let _cfinfo = _CFInfo(typeID: CFAttributedStringGetTypeID())
+    private let _string: NSString
```

Comment:
SR-1129 seems to be a compile time error, what I encountered is a runtime error. 
So, maybe they are not related.